pi.json

```diff
@@ -60,7 +61,11 @@
     "post-processors": [
       {
         "type": "shell-local",
-        "inline": ["xz -vT0 \"{{user `SB_NAME`}}-image-{{user `SB_VERSION`}}.img\""]
+        "inline": [
```

Comment:
Suggestion: Given this isn't really part of the Packer build, perhaps it's worth just extracting it into `build-image.sh` instead.

Reply:
Notably this runs within the docker container so has no requirements on the host system.